static/templates/filings/candidate.hbs

```diff
@@ -56,14 +56,28 @@
       <td class="panel__term">Ending cash on hand</td>
       <td class="panel__data">{{{currency cash_on_hand_end_period}}}</td>
     </tr>
+    {{#if net_contributions_period}}
     <tr>
       <td class="panel__term">Net contributions</td>
       <td class="panel__data">{{{currency net_contributions_period}}}</td>
     </tr>
+    {{else}}
+    <tr>
+      <td class="panel__term">Net contributions</td>
+      <td class="panel__data">{{{currency net_contributions_cycle_to_date}}}</td>
+    </tr>
+    {{/if}}
+    {{#if net_operation_expenditures_period}}
```

Comment:
@jmcarp thanks for the catch. Fixed it.

@LindsayYoung this template is for the reports slide out on the filings view (the screenshot you posted). This is the template that's used for candidate reports. I wrapped it in a conditional so that it will only show if the report doesn't have the `net_contributions_period`